packages/fuchsia_ctl/lib/src/amber_ctl.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:fuchsia_ctl/fuchsia_ctl.dart';
+import 'package:fuchsia_ctl/src/ssh_client.dart';
+import 'package:uuid/uuid.dart';
+
+import 'operation_result.dart';
+
+const SshClient _kSsh = SshClient();
+
+/// Wrapper for amberctl utility for commands executed on the target device.
+@immutable
+class AmberCtl {
```

Comment:
btw, there's a similar wrapper in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/fuchsia/amber_ctl.dart